### ISSUE TITLE
support for multiple thread pings

### ISSUE THREAD

**Issue by Elendiar** [alessandromaggio/pythonping#41]
Problem appears when ping() used with multiple threads. When i need simultaneously ping multiple IPs, i get strange results, like mixed response relative to thread:
```
thread 1 (ip 10.10.10.10) - response for 11.11.11.11
thread 2 (ip 11.11.11.11) - response for 11.11.11.11
thread 3 (ip 12.12.12.12) - response for 10.10.10.10
thread 4 (ip 13.13.13.13) - response for 12.12.12.12
```

After some investigation i find reason for this behavior - `seed_id` in `class Communicator`. In default func `ping()` was no argument for seed, so it always was `self.seed_id = os.getpid() & 0xFFFF` > same for all threads > same identifier for send/response
```
self.send_ping(identifier, seq, payload)
self.responses.append(self.listen_for(identifier, self.timeout))
```

I fix this for me by adding `seed_id` argument and sending random value to `ping()`, but it would be better if it was already provided in the module.

**Comment by malbers15** [comment]
hi @Elendiar ,

How about the following for seed_id in executor.py to be explicit:
`if self.seed_id is None:
self.seed_id = random.randint(0x1, 0xFFFF) & 0xFFFF`

And have you happened to test if a similar update is necessary in the checksum method in icmp.py to identifier as well to make this all thread safe?

**Comment by malbers15** [comment]
Pull request open:
https://github.com/alessandromaggio/pythonping/pull/42

**Comment by alessandromaggio** [comment]
Implemented in release v1.0.11.
